2021-10-28 17:24:11,811 INFO    MainThread:591291 [wandb_setup.py:_flush():69] setting env: {}
2021-10-28 17:24:11,811 INFO    MainThread:591291 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-28 17:24:11,812 INFO    MainThread:591291 [wandb_init.py:_log_setup():348] Logging user logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_172411-1651ypz5/logs/debug.log
2021-10-28 17:24:11,812 INFO    MainThread:591291 [wandb_init.py:_log_setup():349] Logging internal logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_172411-1651ypz5/logs/debug-internal.log
2021-10-28 17:24:11,814 INFO    MainThread:591291 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f09adfdb5b0>
2021-10-28 17:24:11,815 INFO    MainThread:591291 [wandb_init.py:init():381] calling init triggers
2021-10-28 17:24:11,815 INFO    MainThread:591291 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-10-28 17:24:11,815 INFO    MainThread:591291 [wandb_init.py:init():430] starting backend
2021-10-28 17:24:11,815 INFO    MainThread:591291 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 17:24:11,846 INFO    MainThread:591291 [backend.py:ensure_launched():135] starting backend process...
2021-10-28 17:24:11,865 INFO    MainThread:591291 [backend.py:ensure_launched():139] started backend process with pid: 591336
2021-10-28 17:24:11,869 INFO    MainThread:591291 [wandb_init.py:init():435] backend started and connected
2021-10-28 17:24:11,954 INFO    MainThread:591291 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-10-28 17:24:11,958 INFO    MainThread:591291 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-28 17:24:11,958 INFO    MainThread:591291 [wandb_init.py:init():494] updated telemetry
2021-10-28 17:24:11,959 INFO    MainThread:591291 [wandb_init.py:init():517] communicating current version
2021-10-28 17:24:16,960 INFO    MainThread:591291 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-28 17:24:18,162 INFO    MainThread:591291 [wandb_init.py:init():557] starting run threads in backend
2021-10-28 17:24:21,259 INFO    MainThread:591291 [wandb_run.py:_console_start():1605] atexit reg
2021-10-28 17:24:21,260 INFO    MainThread:591291 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-10-28 17:24:21,261 INFO    MainThread:591291 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-10-28 17:24:21,262 INFO    MainThread:591291 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-28 17:24:21,262 INFO    MainThread:591291 [wandb_init.py:init():582] run started, returning control to user process
2021-10-28 17:24:21,265 INFO    MainThread:591291 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:24:21,265 INFO    MainThread:591291 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:24:21,274 INFO    MainThread:591291 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:24:22,612 INFO    MainThread:591291 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:24:22,612 INFO    MainThread:591291 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:25:16,493 INFO    MainThread:591291 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-10-28 17:25:16,494 INFO    MainThread:591291 [wandb_run.py:_restore():1547] restore
2021-10-28 17:25:17,347 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:25:17,803 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:25:18,225 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,333 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,437 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,541 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,652 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,756 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

2021-10-28 17:25:18,860 INFO    MainThread:591291 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10456
}

